plugins/pipelines/openPr.js

```diff
@@ -72,7 +72,7 @@ module.exports = () => ({
                                 throw boom.notImplemented('openPr not implemented for gitlab');
                             }
 
-                            return reply(pullRequest.data.url).code(201);
+                            return reply({ prUrl: pullRequest.data.html_url }).code(201);
```

Comment:
We need to use `html_url` instead of `url` from API response, see reference here: 
`https://developer.github.com/v3/pulls/#create-a-pull-request`

The difference between `url` and `html_url` is that
`url`: `https://api.github.com/repos/adong/sd-open-pr-w-branch/pulls/14`
`html_url`: `https://www.github.com/adong/sd-open-pr-w-branch/pulls/14`